### DISCUSSION TITLE
Add explain query parameter on create / update / delete

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#4827] [Feature Requests]
Allows the comment column in directus_activity to be set on create/update/delete. Allows you to explain your changes like a git commit

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
